environment.yml

```diff
@@ -24,6 +24,8 @@ dependencies:
   - seaborn
   - jupyter
   - fuzzywuzzy
+  - mpi
+  - petsc4py
```

Comment:
Make sure you require >= 3.11, I just got bitten by the silent SeparableObjective -> Residual rename elsewhere...

Reply:
Will do. I'll finish this over the weekend.